src/web/components/SlotBodyEnd.tsx

```diff
@@ -26,18 +26,18 @@ const checkForErrors = (response: any) => {
 type OphanAction = 'INSERT' | 'VIEW';
 
 const epicTestName = 'FrontendDotcomRenderingEpic';
+const epicCampaignCode = 'gdnwb_copts_memco_frontend_dotcom_rendering_epic_dcr';
 
 const sendOphanEvent = (action: OphanAction): void => {
     const componentEvent = {
         component: {
             componentType: 'ACQUISITIONS_EPIC',
             products: ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'],
-            campaignCode:
-                'gdnwb_copts_memco_frontend_dotcom_rendering_epic_dcr',
-            id: 'gdnwb_copts_memco_frontend_dotcom_rendering_epic_dcr',
+            campaignCode: epicCampaignCode,
```

Comment:
Ooh totally minor but if the const was called `campaignCode` you could use the short object key form here.

Reply:
Yeah true I was just being pedantic and wanting the var names to be epic specific